tensorflow/python/keras/_impl/keras/utils/data_utils_test.py

```diff
@@ -160,6 +162,9 @@ def test_generator_enqueuer_threads(self):
     self.assertEqual(len(set(acc) - set(range(100))), 0)
     enqueuer.stop()
 
+  @unittest.skipIf(
+      platform.system() == 'Windows',
```

Comment:
Optional: maybe use `os.name == 'nt'` instead? We're using that elsewhere in the code, and [as far as I can tell](https://stackoverflow.com/a/11672585/3574081) it provides the same information a little bit more cheaply, using a compile-time constant rather than [shelling out to `uname`](https://github.com/python/cpython/blob/5c4b0d063aba0a68c325073f5f312a2c9f40d178/Lib/platform.py#L755) on POSIX-compliant platforms.

Reply:
Done